setup.py

```diff
@@ -7,13 +7,12 @@
     long_description = f.read()
 
 requires = [
-    "torch",
+    "torch>=1.12",
     "numpy",
     "pandas",
     "psutil",
     "threadpoolctl",
-    "scikit-learn>=0.23",
-    "importlib_metadata>=0.7; python_version < '3.8'",
```

Comment:
I see you removed this line. Do you mean to remove support for Python 3.6 or 3.7? It's okay if this is what you want. I just want to confirm it.

Reply:
Yes, I'd be in favor of removing it.